googlemock/CMakeLists.txt

```diff
@@ -37,8 +37,8 @@ endif()
 # as ${gmock_SOURCE_DIR} and to the root binary directory as
 # ${gmock_BINARY_DIR}.
 # Language "C" is required for find_package(Threads).
-project(gmock CXX C)
-cmake_minimum_required(VERSION 2.6.2)
+project(gmock VERSION 1.8.0 LANGUAGES CXX C )
```

Comment:
I just took this from the original project() call. My cmake knowledge is too limited to decide what is redundant here. Is it only the C or because it is included in CXX or are the options given somewhere in hunter?